releases/v2.2.0-alpha.20181119.md

```diff
@@ -0,0 +1,205 @@
+- For compatibility with PostgreSQL, it is once again possible to use the keywords `FAMILY`, `MINVALUE`, `MAXVALUE`, `INDEX`, and `NOTHING` as table names, and the names "index" and "nothing" are once again accepted in the right-hand side of `SET` statement assignments. [#31731][#31731] {% comment %}doc{% endcomment %}
+- Renamed the first column name returned by [`SHOW STATISTICS`](../v2.2/show-statistics.html) to `statistics_name`. [#31927][#31927] {% comment %}doc{% endcomment %}
+- CockroachDB now accepts a wider variety of date, time, and timestamp formats. [#31758][#31758] {% comment %}doc{% endcomment %}
+- The new `experimental_vectorize` [session setting](../v2.2/set-vars.html), when enabled, causes columnar operators to be planned instead of row-by-row processors, when possible. [#31354][#31354] {% comment %}doc{% endcomment %}
+- CockroachDB now supports the `BIT` and `VARBIT (BIT VARYING)` bit array data types like PostgreSQL. Currently, only the bit array literal notation with a capital B (e.g., `B'10001'`) is supported; the notation with a small `b` (e.g., `b'abcd'`) continues to denote **byte** arrays as in previous versions of CockroachDB. [#28807][#28807] {% comment %}doc{% endcomment %}
+- Added the `array_to_json` [built-in function](../v2.2/functions-and-operators.html). [#29818][#29818]
+
+### Command-line changes
+
+- The [`cockroach cert create-client`](../v2.2/create-security-certificates.html) now offers the `--also-generate-pkcs8-key` flag for writing a client key in PKCS#8 format. [#29008][#29008] {% comment %}doc{% endcomment %}
+- The client-side option `smart_prompt` now controls whether [`cockroach sql`](../v2.2/use-the-built-in-sql-client.html) and [`cockroach demo`](../v2.2/cockroach-demo.html) use the current transaction state to offer a multi-line entry at the start of new transactions. [#31630][#31630] {% comment %}doc{% endcomment %}
+- The [`cockroach sql`](../v2.2/use-the-built-in-sql-client.html) and [`cockroach demo`](../v2.2/cockroach-demo.html) commands now recognize the commands `exit` and `quit` to terminate the shell. [#31915][#31915] {% comment %}doc{% endcomment %}
+- The `cockroach debug estimate-gc` command now allows users to specify TTL period, with a default of 24 hours. [#31402][#31402]
+
+### Admin UI changes
+
+- Improved the layout of the [**Cluster Overview**](../v2.2/admin-ui-cluster-overview-page.html) page for large clusters with many nodes and ranges. [#31512][#31512] {% comment %}doc{% endcomment %}
+- Added the current node ID to the [**Advanced Debugging**](../v2.2/admin-ui-debug-pages.html) page to help identify the current node when viewing the web UI through a load balancer. [#31835][#31835] {% comment %}doc{% endcomment %}
+- The "Non-Table Cluster Data" section of the [**Databases**](../v2.2/admin-ui-databases-page.html) page now includes all non-table data types. Previously, this section only showed Time Series data. [#31830][#31830] {% comment %}doc{% endcomment %}
```

Comment:
"Non-Table Cluster Data" section should be **Non-Table Cluster Data** section, per our Style Guide

Reply:
Done.